### PR TITLE
Add denoising autoencoder in the model zoo

### PR DESCRIPTION
**Pull request by haabhi**
Import works fine. Export for keras and tensorflow works smoothly, but caffe does not support GaussianNoise layer so the model cannot be exported for caffe.

### CODE REVIEW

**Comment by Ram81** [comment]
@haabhi can you share a screenshot of imported model

**Comment by gautamjajoo** [comment]
@haabhi And add it to [Tested models](https://github.com/Cloud-CV/Fabrik/blob/master/tutorials/tested_models.md) too.

**Comment by haabhi** [comment]
@Ram81 Here is the screenshot of the model.
imported-model
And the model zoo:
model-zoo

**Comment by Ram81** [comment]
@haabhi can you also add the model in tested models and check whether export works for caffe, keras and tensorlow

**Comment by haabhi** [comment]
@Ram81 I've added the model in tested models https://github.com/Cloud-CV/Fabrik/pull/413/commits/7a699e65850159d8b9c53bef8d8166fdcf5d63ae
And export works for both tensorflow and keras. Caffe does not support GaussianNoise and Upsample layers, so export for caffe does not work.
thanks :)

**Comment by Ram81** [comment]
@haabhi I meant you have to add in the table in `readme.md` and list down there for which framework import and export work

**Comment by haabhi** [comment]
@Ram81 done :+1:

File: README.md [hunk 1]
```diff
@@ -290,7 +290,7 @@ Models                                                                      | Ca
 [YoloNet](https://arxiv.org/abs/1506.02640)                                 |   √   |   √   |     √      |
 [Pix2Pix](https://github.com/phillipi/pix2pix)                              |   √   |   ×   |     ×      |
 [VQA](https://github.com/iamaaditya/VQA_Demo)                               |   √   |   √   |     √      |
-
+[Denoising Auto-Encoder](https://blog.keras.io/building-autoencoders-in-keras.html)                               |   ×   |   √   |     √      |
 Note: For models that use a custom LRN layer (Alexnet), Keras expects the custom layer to be passed when it is loaded from json. LRN.py is located in keras_app/custom_layers. [Alexnet import for Keras](https://github.com/Cloud-CV/Fabrik/blob/master/tutorials/keras_custom_layer_usage.md)
```
Comment (Ram81) [inline README.md]: Can you move this out of the table, I know it is not related your task but please move it out of the table.

**Comment by haabhi** [comment]
@Ram81 A new line between the table and the note is needed right?

**Comment by Ram81** [comment]
Yes, LGTM 👍

**Pull request merged**